recruiter/forms.py

```diff
@@ -0,0 +1,29 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
+from .models import Recruiter, Company
+
+
+class recuiterRegistrationForm(UserCreationForm):
```

Comment:
Please follow the usual Python Pep8 styling rules - class names should begin with upper case characters.
Not sure why flake8 is letting this pass.

Reply:
will be fixed if this is the convention.